docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -248,6 +248,12 @@ Describe table API:
 
 - **source_type:** select the configured databases (e.g. `hive`) or connector ID (e.g. `1`)
 
+Analyze API:
+
+    curl -X POST 'https://demo.gethue.com/api/<DIALECT>/analyze/<DB>/<TABLE>/' -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNjI1ODE1MzU5LCJqdGkiOiIzODM0M2Q3YjRjNGY0NTUxYWJmYmQyZmI4YjUzYjFjZiIsInVzZXJfaWQiOjExMDA3MTR9.fl0h7VooLtWnu9v7FtdLUy3NukwFtUya-LkTzollTig' -H 'Content-Type: application/x-www-form-urlencoded' -d ''
```

Comment:
`-d ''` needed? (we could just have a default value if really needed)

Reply:
Removed